apps/browser/qml/pages/components/AddToAppGridDialog.qml

```diff
@@ -0,0 +1,16 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+BookmarkEditDialog {
+    property string icon
+    property var desktopBookmarkWriter
+    property var view
```

Comment:
```suggestion
    property QtObject bookmarkWriterParent
```

This is in reality an owner / parent of the to be created desktopBookmarkWriter not a view per se. Hence, above suggestion.